README.rst

```diff
@@ -103,130 +103,132 @@ Functionality Overview
-allows to significantly reduce transmission tail latencies for applications and to
-avoid locking under multi-queue NICs compared to traditional approaches such as HTB
-(Hierarchy Token Bucket) or TBF (Token Bucket Filter) as used in the bandwidth CNI
-plugin, for example.
-
-Monitoring and Troubleshooting
-------------------------------
-
-The ability to gain visibility and troubleshoot issues is fundamental to the
-operation of any distributed system. While we learned to love tools like
-``tcpdump`` and ``ping`` and while they will always find a special place in our
-hearts, we strive to provide better tooling for troubleshooting. This includes
-tooling to provide:
-
-- Event monitoring with metadata: When a packet is dropped, the tool doesn't
-  just report the source and destination IP of the packet, the tool provides
-  the full label information of both the sender and receiver among a lot of
-  other information.
-
-- Metrics export via Prometheus: Key metrics are exported via Prometheus for
-  integration with your existing dashboards.
-
-- Hubble_: An observability platform specifically written for Cilium. It
-  provides service dependency maps, operational monitoring and alerting,
-  and application and security visibility based on flow logs.
-
-.. _Hubble: https://github.com/cilium/hubble/
-.. _`Layer 7 Policy`: https://docs.cilium.io/en/stable/security/policy/language/#layer-7-examples
+containers and to/from external services. The load balancing is implemented
+in eBPF using efficient hashtables allowing for almost unlimited scale.
+
+* **East-west load balancing** rewrites service connections at the socket
+  level (``connect()``), avoiding the overhead of per-packet NAT and fully
+  `replacing kube-proxy <https://cilium.io/use-cases/kube-proxy/>`_.
+
+* **North-south load balancing** supports XDP for high-throughput scenarios
+  and `layer 4 load balancing <https://cilium.io/use-cases/load-balancer/>`_
+  including Direct Server Return (DSR), and Maglev consistent hashing.
+
+Cluster Mesh
+------------
+
+Cilium `Cluster Mesh <https://cilium.io/use-cases/cluster-mesh/>`_ enables
+secure, seamless connectivity across multiple Kubernetes clusters. For
+operators running hybrid or multi-cloud environments, Cluster Mesh ensures
+a consistent security and connectivity experience.
+
+* **Global service discovery**: Workloads across clusters can discover and
+  connect to services as if they were local. This enables fault tolerance,
+  like automatically failing over to backends in another cluster, and
+  exposes shared services like logging, auth, or databases across
+  environments.
+
+* **Unified identity model:** Security policies are enforced based on
+  identity, not IP address, across all clusters.
+
+Network Policy
+--------------
+
+Cilium `Network Policy <https://cilium.io/use-cases/network-policy/>`_
+provides identity-aware enforcement across L3--L7. Typical container
```

Comment:
GitHub doesn't seem to change the double-dash to an en-dash in `L3--L7`, which probably doesn't matter much. You could use just a single dash instead, or a Unicode en-dash directly, or an HTML entity: `&ndash;` &ndash;